data/json/items/comestibles/egg.json

```
@@ -129,14 +128,15 @@
     "proportional": { "volume": 1.4, "weight": 1.4 },
     "calories": 130,
     "vitamins": [ [ "iron", "3 mg" ], [ "calcium", "45 mg" ], [ "egg_allergen", 1 ] ],
-    "rot_spawn": "GROUP_EGG_DUCK"
+    "rot_spawn": { "group": "GROUP_EGG_DUCK" }
   },
   {
     "type": "COMESTIBLE",
     "id": "egg_duck_domestic",
     "name": { "str": "domestic duck egg" },
     "copy-from": "egg_duck",
-    "rot_spawn": "GROUP_EGG_DUCK_DOMESTIC"
+    "spoils_in": "20 minutes",
```

```suggestion
    "spoils_in": "20 minutes",
```

Nuh huh. That's your testing changes you never reverted.